sympy/calculus/singularities.py

```diff
@@ -239,14 +208,15 @@ def is_monotonic(f, interval=S.Reals, symbol=None):
 
     """
     from sympy.core.logic import fuzzy_or
-    f = sympify(f)
-    free_sym = f.free_symbols
-
-    if symbol is None and len(free_sym) > 1:
-        raise NotImplementedError('is_monotonic has not yet been '
-                                'for all multivariate expressions')
-
-    inc = is_increasing(f, interval, symbol)
-    dec = is_decreasing(f, interval, symbol)
-
-    return fuzzy_or([inc, dec])
```

Comment:
Why do we use `fuzzy_or` instead of returning `inc or dec`? Both `inc` and `dec` are booleans, so it should work; also all tests are being passed when returning `inc or dec`.